src/index.js

```diff
@@ -42,5 +45,24 @@ ul2.addEventListener('click', (event) => {
     const itemKey = [...added].indexOf(parent) + 1;
     removeItem(itemKey);
   }
+  if (event.target.id === 'checkbox') {
+    const checkbox = event.target;
+    const added = document.querySelectorAll('.added');
+    const parent = checkbox.parentElement.parentElement;
+    const itemKey = [...added].indexOf(parent);
+    toDoData[itemKey].completed = checkbox.checked;
+    checkbox.addEventListener('change', () => {
+      displayList(toDoData);
+      isChecked(toDoData);
+      setItems(toDoData);
+    });
```

Comment:
Well done overall on the project 💯 everything is working perfectly but there's a minor issue when I check a todo ☑️ the `todo` item changes the order of the list. See below, I created three todo items and checked two of them 👇🏽 

![Screenshot 2023-03-03 at 01 06 10](https://user-images.githubusercontent.com/45739963/222644140-c5a84959-0c36-4bc7-b9ef-5854037c8d4a.png)

Look to address that as this is the last thing that needs some work 😉